src/.prettierrc

```diff
@@ -0,0 +1,11 @@
+{
```

Comment:
Why have we got this file in the PR? It does not seem part of the ticket.

Reply:
It is in due to develop rebasing after ADR merge.